src/components/manage/Sidebar/ObjectBrowserNav.jsx

```diff
@@ -90,7 +90,13 @@ const ObjectBrowserNav = ({
               {item.title}
             </span>
             {item.is_folderish && mode === 'image' && (
-              <Icon name={rightArrowSVG} size="24px" />
+              <span>
```

Comment:
@iFlameing was this span element added by accident? If we change the HTML structure we might have to consider this as a breaking change. I don't see the use of this span element at first, could you elaborate?

Reply:
+1 seems unrelated or a left over?

Reply:
sorry!  I added this to increase the hover effect's space, but it doesn't look good as per UI and I forget to remove it. I will fix it today.